packages/components/src/promo-card/index.tsx

```diff
@@ -0,0 +1,30 @@
+import './style.scss';
+import { AppStoreSVG, JetpackSVG, QRCodeSVG, WordPressSVG } from './svg-icons';
+
+export type PromoCardProps = {
+	data: {
+		title: string | null;
+		message: string | null;
+		cta?: string;
```

Comment:
Do you think there will be more `cta` options available? You can make this type more specific to accept only a concrete string or a set of strings.
```suggestion
		cta?: 'app-store';
```

Reply:
I do not foresee other CTA options so I'm guessing it probably makes more sense as a boolean value. That said, I'm a bit unclear on the intended logic here. The design calls for a QR code on larger viewports and the Apple app store graphic on mobile viewports. I'm not sure it makes sense to have an image/link to the Apple app store though. Might make more sense to have a custom image that just links to wp.com/app so they get the correct app store for their device.

I'll make this a boolean value for now and seek some design feedback in the Issue.